### ISSUE TITLE
schema:publishingPrinciples or schema:variableMeasured for EOVs?

### ISSUE THREAD

**Issue by pieterprovoost** [marco-bolo/provenance-sandbox#12]
`schema:variableMeasured` are typically more granular than the high level EOVs, and probably correspond better to EOV subvariables. I believe @pbuttigieg suggested to use `schema:publishingPrinciples` to point to the EOV spec sheets. URIs are not available yet.

```json
{
    // ...
    "publishingPrinciples": [
        {
            "@id": "https://www.goosocean.org/eov/microbial_biomass_diversity",
            "@type": "CreativeWork",
            "name": "Microbial biomass and diversity EOV",
            "author": {
                "@type": "Organization",
                "@id": "https://goosocean.org",
                "name": "GOOS"
            }
        }
    ]
}
```

**Comment by pbuttigieg** [comment]
>I believe @pbuttigieg suggested to use schema:publishingPrinciples to point to the EOV spec sheets. URIs are not available yet.

@pieterprovoost yes, the `publishingPrinciples` would be the spec sheet if - and only if - the digital asset it describes really was guided by that spec sheet. 

I hope we can get those spec sheets into AquaDocs for DOI creation. In the meantime, we can add the `url` that links to the sheet, e.g. https://oceanexpert.org/downloadFile/57795 

I'm surprised to see some in Ocean Expert. I still don't consider these PIDs formally, but that's better than a wild URL on GOOS website.

>schema:variableMeasured are typically more granular than the high level EOVs, and probably correspond better to EOV subvariables.

This is true, but we've now unpacked the aggregate terms we can use them as `propertyID` values in `variableMeasured`. xref https://github.com/marco-bolo/dataset-catalogue/issues/15

Many of those overlap with the subvariables, but the rubric for the latter is unclear. We can create ENVO classes for those, if desired.
